corehq/apps/locations/tests/util.py

```diff
@@ -28,11 +25,6 @@ def make_loc(code, name=None, domain=TEST_DOMAIN, type=TEST_LOCATION_TYPE,
 
 @unit_testing_only
 def delete_all_locations():
-    ids = [
-        doc['id'] for doc in
-        SupplyPointCase.get_db().view('supply_point_by_loc/view', reduce=False).all()
-    ]
-    iter_bulk_delete(SupplyPointCase.get_db(), ids)
```

Comment:
should this not delete the SQL cases that get created instead?

Reply:
Oh, hmm. Maybe, although none of the tests that use this are `@sharded`, so the transaction rollback should take care of that automatically. I'm hesitant to add another "delete all the things" call here since that is a testing antipattern that we should be moving away from when possible.